service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GpcTemplateUtils.java

```diff
@@ -0,0 +1,33 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+import com.github.mustachejava.DefaultMustacheFactory;
+import com.github.mustachejava.Mustache;
+import com.github.mustachejava.MustacheFactory;
+
+import java.io.IOException;
+import java.io.StringWriter;
+
+public class GpcTemplateUtils {
+
+    private static final String TEMPLATES_DIRECTORY = "templates";
+
+    public static Mustache loadTemplate(String templateName) {
+        MustacheFactory mf = new DefaultMustacheFactory(TEMPLATES_DIRECTORY);
+        Mustache m = mf.compile(templateName);
+        return m;
+    }
+
+    public static String fillTemplate(Mustache template, Object content) {
+        StringWriter writer = new StringWriter();
+        String data = "";
+
+        try {
+            template.execute(writer, content).flush();
+            data += writer.toString();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
It is not a good practice to print a stack trace. Try making an error log, please.